cfme/fixtures/v2v.py

```diff
@@ -163,32 +165,28 @@ def get_vm(request, appliance, source_provider, template, datastore="nfs"):
     return vm_obj
 
 
-def _form_data(source_provider, provider):
-    form_data = {
-        "general": {
-            "name": "infra_map_{}".format(fauxfactory.gen_alphanumeric()),
-            "description": "Single Datastore migration of VM from {ds_type1} to"
-            " {ds_type2}".format(ds_type1="nfs", ds_type2="nfs"),
-        },
-        "cluster": {"mappings": [_form_data_mapping("clusters", source_provider, provider)]},
-        "datastore": {
-            "Cluster ({})".format(provider.data.get("clusters")[0]): {
-                "mappings": [
-                    _form_data_mapping("datastores", source_provider, provider, "nfs", "nfs")
-                ]
-            }
-        },
-        "network": {
-            "Cluster ({})".format(provider.data.get("clusters")[0]): {
-                "mappings": [
-                    _form_data_mapping(
-                        "vlans", source_provider, provider, "VM Network", "ovirtmgmt"
-                    )
-                ]
-            }
-        },
+def infra_mapping_default_data(source_provider, provider):
+    """
+    Default data for infrastructure mapping form.
+    It is used in other methods to recursive update the data accoriding
+    to parameters in tests.
+
+    :param source_provider: Vmware provider
+    :param provider: Target rhev/OSP provider
+    :return:
```

Comment:
I like this doc string style helped lot in doc building... but we are using different...
https://gist.github.com/mfalesni/11694e0d563c80c3917ce6012400b93f
Its old please correct me if missing something.

Reply:
yes that was default generated doc string